workspace/app/app.js

```diff
@@ -132,4 +138,6 @@ module.run(function (ImpacLinking, ImpacAssets, ImpacRoutes, ImpacTheming, Impac
   // -------------------------------------------------------
   ImpacLinking.linkData(DevSettings.buildLinkingConfig(defaults.orgUid, defaults.mnoeUrl));
 
+  // Locale to be used
+  $translate.use('en-AU');
```

Comment:
Everything that's in `/workspace` is just the developer interface, it's like an application that integrates impac-angular, so the result would be the same